### ISSUE TITLE
Bug: cannot find name 'fastjsDom' after choosing fastjs with TypeScript

### ISSUE THREAD

**Issue by Rotten-LKZ** [fastjs-team/fastjs-cli#7]
**Describe the bug**
![Screenshots](https://user-images.githubusercontent.com/54099603/210130683-fe6d576c-2f4d-49b8-902e-24e455b14229.png)
Editor can't find the type `fastjsDom`

**To Reproduce**
1. Use `fastjs create` and choose fastjs with TypeScript
2. Open file `main.ts`
3. The error occures

**Expected behavior**
No error

**Screenshots**
![Screenshots](https://user-images.githubusercontent.com/54099603/210130683-fe6d576c-2f4d-49b8-902e-24e455b14229.png)

**Additional context**
1. The editor can infer the type returned by the `selecter` itself. Therefore, the type declaration of this variable is redundant.
2. If you want to let us know that the project is under `TypeScript`, here are two possible solutions.
```typescript
import type { FastjsDom, FastjsDomList } from 'fastjs-next'

// ...

let el: FastjsDom | FastjsDomList = selecter("#counter");
```
Or
```typescript
let el: ReturnType<typeof selecter> = selecter("#counter"); // This seems to be a bad solution
```

**Version**
![image](https://user-images.githubusercontent.com/54099603/210130289-5fd5a845-0b91-44f2-9c0a-4bfe9fd80172.png)
Node.js: v18.12.1
fastjs-cli: v2.4.3

**Comment by xiaodong2008** [comment]
Hi, this problem is in todo list now, thanks for your issue!

**Comment by xiaodong2008** [comment]
This problem has been fixed in commit dda0a9aecc9294b351f90be065ebb8e643fa0e77
And it will be publish at release v2.4.4
